acasclient/acasclient.py

```diff
@@ -1930,6 +1930,32 @@ def _validate_then_save_codetable(self, url_base, codeTable: dict) -> dict:
         resp.raise_for_status()
         return resp.json()
     
+    def get_assay_scientists(self):
```

Comment:
Main changes for adding assay scientists, these functions are very similar to the cmpdreg scientists.